Source/WebCore/platform/graphics/cocoa/SystemFontDatabaseCoreText.cpp

```diff
@@ -392,7 +393,7 @@ auto SystemFontDatabase::platformSystemFontShorthandInfo(FontShorthand fontShort
 
     auto interrogateTextStyleShorthandItem = [] (CFStringRef textStyle) {
         CGFloat weight = 0;
-        float size = CTFontDescriptorGetTextStyleSize(textStyle, contentSizeCategory(), kCTFontTextStylePlatformDefault, &weight, nullptr);
+        float size = CTFontDescriptorGetTextStyleSize(textStyle, contentSizeCategory(), PAL::currentUserInterfaceIdiomIsReality() ? kCTFontTextStylePlatformDefault : kCTFontTextStylePlatformPhone, &weight, nullptr);
```

Comment:
This seems wrong: on *all* non-visionOS platforms you're going to say kCTFontTextStylePlatformPhone

Reply:
And with the minor complexity that fixing that is going to add, can you maybe shuffle it out to a static function outside of this one?